pkg/crc/network/proxy.go

```diff
@@ -0,0 +1,143 @@
+package network
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/asaskevich/govalidator"
+)
+
+var defaultNoProxies = []string{".apps-crc.testing", "127.0.0.1", "api.crc.testing", "localhost"}
```

Comment:
`apps-crc.testing` and `crc.testing` should be coming from the bundle, not be hardcoded here.